packages/odyssey-react/src/components/Tag/Tag.tsx

```diff
@@ -43,16 +36,12 @@ export const Tag = withTheme(
   styles
 )(
   forwardRef<HTMLUListElement, TagProps>((props, ref) => {
-    const { tags, ...rest } = props;
+    const { children, ...rest } = props;
     const omitProps = useOmit(rest);
 
     return (
-      <Box as="ul" {...omitProps} ref={ref} className={styles.root}>
-        {tags.map((item) => (
-          <li className={styles.item} key={item}>
-            {item}
-          </li>
```

Comment:
@francoislehoux-okta noticing a gap in our previous discussions. I'm realizing now that there **is** semantic meaning here in the rendered markup because this component will describe one or more tags as a list. This is something we'd want to maintain rather than having downstream consumers rebuild this themselves. So, I think wall we need here is perhaps a rename of the component itself, something like `TagList` - thoughts?

Reply:
I don't mind reverting this back and going with the rename approach, hat's what I initially wanted to go with.

Do we need a discussion around renaming to `TagList` vs Tags`?

Reply:
Do we consider the current `Tabs` component to be a `list/item` component? Trying to keep everything as coherent as possible

Reply:
Perfect, reverting. Will push in a few :)